config/sections/mixins/pagination.php

```diff
@@ -14,7 +14,7 @@
          * Sets the default page for the pagination. This will overwrite default pagination.
          */
         'page' => function (int $page = null) {
-            return get('page', $page ?? 1);
+            return get('page', $page);
```

Comment:
Is there a special reason not to fall back to 1 as the default page?

Reply:
The automatic fallback now happens directly in the `setProperties()` method. The reason we need to do it there is that it would otherwise throw an Exception if the default page of 1 is passed for a collection with 0 items (= 0 pages).